src/components/CookieConsent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import 'vanilla-cookieconsent/dist/cookieconsent.css';
+import * as CookieConsent from 'vanilla-cookieconsent';
+import LeadinfoScript from '../components/layout/leadinfo-script';
+
+const CookieConsentComponentV3 = () => {
+  const [analyticsEnabled, setAnalyticsEnabled] = useState(false);
+
+  useEffect(() => {
+    CookieConsent.run({
+      categories: {
+        necessary: {
+          enabled: true,
+          readOnly: true,
+        },
```

Comment:
we don't need that category, do we ? Seems like you copied that as seen in the docs?